.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -27,12 +27,8 @@ body:
       description: Which server version version you using? If your server version is not listed, it is not supported. Update to a supported version first.
       multiple: false
       options:
-        - '1.20'
+        - '1.20.1'
         - '1.19.4'
-        - '1.19.3'
-        - '1.19.2'
-        - '1.19.1'
-        - '1.19'
         - '1.18.2'
         - '1.17.1'
         - '1.16.5'
```

Comment:
You should probably remove 1.16.5 from here too.

Reply:
Good catch!